Decisions.Jira.sln

```diff
@@ -3,7 +3,6 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29911.84
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Decisions.Jira", "Decisions.Jira\Decisions.Jira.csproj", "{B782A73F-DB60-4A66-8522-5A8849A0F2D0}"

```

Comment:
This removes the main project, but keeps the TestSuite project.  If we are going to keep the .sln file for customers to see, it should contain everything buildable in the module.  If the .sln is no longer needed, please delete it.